ActionMenu/UIResources/actionmenu.json

```diff
@@ -1,162 +1,209 @@
-                    "type": "system call",
-                    "event": "ShowMainMenuPage",
-                    "event_arguments": [ "settings" ]
-                }
-            },
-            {
-                "name": "Action Menu",
-                "icon": "icon_actionmenu.svg",
-                "action": {
-                    "type": "menu",
-                    "menu": "ActionMenu/settings"
-                }
-            }
-        ],
-        "avatar": [
-            {
-                "name": "Emotes",
-                "icon": "icon_avatar_emotes.svg",
-                "action": {
-                    "type": "menu",
-                    "menu": "avatar/emotes"
-                }
-            },
-            {
-                "name": "Spray",
-                "action": {
-                    "type": "avatar parameter",
-                    "parameter": "Spray",
-                    "control": "toggle",
-                    "value": 1
-                }
-            },
-            {
-                "name": "Emoji Fox",
-                "icon": "icon_emoji_fox.png",
-                "action": {
-                    "type": "avatar parameter",
-                    "parameter": "Emoji",
-                    "control": "impulse",
-                    "value": 1,
-                    "duration": 0.5
-                }
-            },
-            {
-                "icon": "icon_avatar_ears.png",
-                "action": {
-                    "type": "avatar parameter",
-                    "parameter": "GestureRightWeight",
-                    "control": "radial",
-                    "default_value": -0.5,
-                    "min_value": -1,
-                    "max_value": 1
-                }
-            },
-            {
-                "name": "Joystick 2D",
-                "action": {
-                    "type": "avatar parameter",
-                    "parameter": "testjoystick2d",
-                    "control": "joystick_2d",
-                    "default_value_x": 0,
-                    "min_value_x": 0,
-                    "max_value_x": 0.999,
-                    "default_value_y": 0,
-                    "min_value_y": 0,
-                    "max_value_y": 0.999
-                }
-            },
-            {
-                "name": "Input 2D",
-                "action": {
-                    "type": "avatar parameter",
-                    "parameter": "testjoystick2d",
-                    "control": "input_vector_2d",
-                    "default_value_x": 0,
-                    "min_value_x": 0,
-                    "max_value_x": 0.999,
-                    "default_value_y": 0,
-                    "min_value_y": 0,
-                    "max_value_y": 0.999
-                }
-            }
-        ]
-    }
+  "menus": {
+    "main": [
+      {
+        "name": "Actions",
+        "icon": "icon_actionmenu.svg",
+        "action": {
+          "type": "menu",
+          "menu": "actions"
```

Comment:
As discussed about the action menu itself, it can be customized through json overrides. I think everything in here can be achieved with it because all the Quick Menu function are available, you can use them.

Also I don't think those are "Actions". I will probably not merge this unless many people think this configuration is preferrable. People are free to upvote the PR to show interest.